File: webshop/src/app/shared/modelsAndTheirServices/animator.ts
export class Animator {

  private animatorID;
  private animatorName;
  private animatorLink;
  private animatorImage;

  constructor(private data?) {
    this.animatorID = data.animatorID;
    this.animatorName = data.animatorName;
    this.animatorLink = data.animatorLink;
    this.animatorImage = data.animatorImage;
  }

  public getData() {
    const data = {
      animatorID: this.animatorID,
      animatorName: this.animatorName,
      animatorLink: this.animatorLink,
      animatorImage: this.animatorImage
    };
    return data;
  }

  public getAnimatorID() {
    return this.animatorID;
  }

  public getAnimatorName() {
    return this.animatorName;
  }

  public getAnimatorLink() {
    return this.animatorLink;
  }

  public getAnimatorImage() {
    return this.animatorImage;
  }
}

File: src/main/java/nl/hsleiden/model/Animator.java
package nl.hsleiden.model;

public class Animator {
    private int animatorID;
    private String animatorName;
    private String animatorLink;
    private String animatorImage;

    public Animator() {

    }

    public Animator(int animatorID, String animatorName, String animatorLink, String animatorImage) {
        this.animatorID = animatorID;
        this.animatorName = animatorName;
        this.animatorLink = animatorLink;
        this.animatorImage = animatorImage;
    }

    public int getAnimatorID() {
        return animatorID;
    }

    public void setAnimatorID(int animatorID) {
        this.animatorID = animatorID;
    }

    public String getAnimatorName() {
        return animatorName;
    }

    public void setAnimatorName(String animatorName) {
        this.animatorName = animatorName;
    }

    public String getAnimatorLink() {
        return animatorLink;
    }

    public void setAnimatorLink(String animatorLink) {
        this.animatorLink = animatorLink;
    }

    public String getAnimatorImage() {
        return animatorImage;
    }

    public void setAnimatorImage(String animatorImage) {
        this.animatorImage = animatorImage;
    }
}
